crd-docs/cr/kiali.io_v1alpha1_kiali.yaml

```diff
@@ -354,6 +354,8 @@ spec:
     # default: private_key_file is undefined
     private_key_file: ""
 
+  istio_api_enabled: true
```

Comment:
this doesn't seem to need to be a whole new top-level section. This should go under `external_services.istio` with the rest of the istio-specific configuration settings.

Reply:
Thanks for your feedback, @jmazzitelli ! I will push a new commit.